lib/ddtrace/contrib/rails/active_support/callbacks/rails50.rb

```diff
@@ -0,0 +1,77 @@
+module Datadog
+  module Contrib
+    module Rails
+      module ActiveSupport
+        module Callbacks
+          # Callbacks module specifically for Rails 5.0
+          module Rails50
+            # Module for patching the callback chain
+            module CallbackChain
+              def compile
+                # TODO: Optimize this... don't loop every time callbacks are compiled.
```

Comment:
since the impact is in any request, we should avoid actions that can introduce useless overhead

Reply:
It's not overhead exactly... it's just that we have to do it on each request if we can't patch callbacks before requests come in. This comment was more to say, "if we can do this in a more optimized way, don't do this, do something better."